src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -1153,6 +1158,17 @@ def blobs_bucket():
   return local_config.ProjectConfig().get('blobs.bucket')
 
 
+def use_async_http():
+  global _USES_ASYNC_HTTP
```

Comment:
nit: just use `memoize` here. 

e.g. https://github.com/google/clusterfuzz/blob/2d57351ad8e22597a94849db6e013b12d9dcdc3c/src/clusterfuzz/_internal/base/utils.py#L904

Reply:
I couldn't figure out if it works with no arguments and it seems a little less readable to be honest. I'll switch it if you think it works.

Reply:
it should work -- the example linked has no arguments.